packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -82,17 +85,25 @@ function UncontrolledInnerBlocks( props ) {
 		[ clientId ]
 	);
 
-	const { allowSizingOnChildren = false } =
-		getBlockSupport( name, '__experimentalLayout' ) || {};
+	const defaultLayoutBlockSupport =
+		getBlockSupport( name, '__experimentalLayout' ) || EMPTY_OBJECT;
+
+	const { allowSizingOnChildren = false } = defaultLayoutBlockSupport;
+
+	const defaultLayout = useSetting( 'layout' ) || EMPTY_OBJECT;
 
-	const layout = useMemo(
+	const usedLayout = layout || defaultLayoutBlockSupport;
+
+	const memoedLayout = useMemo(
 		() => ( {
-			...__experimentalLayout,
+			// Default layout will know about any content/wide size defined by the theme.
+			...defaultLayout,
```

Comment:
At the moment, expanding `...defaultLayout` gives the passed down layout object access to the theme's content/wide size, so I think this sounds pretty good to me. But, looking at this raises a couple of potential questions (not blockers, just questions):

* Will the child always want content/wide size from the theme? Or to put it differently, will children ever need to know where the value comes from (i.e. parent vs theme)
* Will the child always want all the attributes from the default layout object, or is it just the content/wide size? Settings objects don't always have the same shape as objects in block attributes... but maybe they should match? If we're concerned about this, one option might be to just pick content/wide size here rather than passing everything to the `memoedLayout`.
* The default layout object also includes layout definitions, which could be really useful to child, or might not be needed potentially. Does it matter that this is merged into `memoedLayout`? I don't have a strong opinion either way here, mostly just curious if it's something we'll need to keep in mind.

Reply:
Good questions!

> Will the child always want content/wide size from the theme? Or to put it differently, will children ever need to know where the value comes from (i.e. parent vs theme)

If the parent also has a content or wide size set, it will override the theme one because we're adding `usedLayout` under `defaultLayout`.

> Will the child always want all the attributes from the default layout object, or is it just the content/wide size?

> The default layout object also includes layout definitions, which could be really useful to child, or might not be needed potentially.

We don't know what we might yet need 😄 but given we're passing layout definitions around pretty much everywhere in layout, this just sticks to the same behaviour. The general idea is that `BlockListItems` should be able to access anything from layout.

Reply:
> If the parent also has a content or wide size set, it will override the theme one because we're adding usedLayout under defaultLayout.

Yeah, I think what I was curious about was whether or not the child will ever need to know where it came from (if the value was explicitly set on the parent, or if it's inherited from the default layout object). It's probably not something to worry about though, because we intentionally removed that complexity by getting rid of the `inherit` prop. I imagine it'll only be something to look into if we ever have a use case for it, which seems unlikely 🙂

> We don't know what we might yet need 😄 but given we're passing layout definitions around pretty much everywhere in layout, this just sticks to the same behaviour. 

Sounds good. And if we stick to passing everything down, then we've got consistent behaviour for if/when additional settings get added in `theme.json`, so it shouldn't be a surprise 👍

I'll resolve this thread.